modules/masterbar/admin-menu/admin-menu.css

```diff
@@ -249,48 +232,503 @@
+	border-bottom: 1px solid #26292c; /* Calypso --theme-submenu-background-color */
+	border-top: 1px solid #26292c; /* Calypso --theme-submenu-background-color */
+}
+
+.admin-color-midnight .site__info .site__title,
+.admin-color-midnight .site__info .site__domain {
+	color: #ffffff !important; /* Calypso --theme-text-color */;
+}
+
+.admin-color-midnight #adminmenu .toplevel_page_site-card:hover div.wp-menu-image,
+.admin-color-midnight #adminmenu .toplevel_page_site-card a:focus div.wp-menu-image {
+	background-color: #e14d43; /* Calypso --theme-highlight-color */
+}
+
+.admin-color-midnight .site__info .site__title::after,
+.admin-color-midnight .site__info .site__domain::after {
+	background: linear-gradient(90deg,rgba(54, 59, 63,0),rgba(54, 59, 63,1) 90%); /* Calypso --theme-base-color-rgb */
+}
+
+.admin-color-midnight .site__info > .site__badge {
+	background: #e14d43; /* Calypso --theme-highlight-color */
+	color: #fff; /* Calypso --theme-text-color */
+}
+
+/* Ocean */
+
+/* Masterbar */
+.admin-color-ocean #wpadminbar {
+	-webkit-box-shadow: inset 0 -1px 0 #627c83;
+    -moz-box-shadow: inset 0 -1px 0 #627c83;
+    box-shadow: inset 0 -1px 0 #627c83; /* Calypso --theme-submenu-background-color */
+}
+
+.admin-color-ocean #wpadminbar .ab-top-menu > #wp-admin-bar-blog > .ab-item {
+	background: #627c83; /* Calypso --theme-submenu-background-color */
+	color: #ffffff !important; /* Calypso --theme-text-color */
+}
+
+.admin-color-ocean #wpadminbar .ab-top-menu > #wp-admin-bar-newdash > .ab-item {
+	color: #ffffff !important; /* Calypso --theme-text-color */
+}
+
+.admin-color-ocean #wpadminbar:not(.mobile) .ab-top-menu > li:hover > .ab-item {
+	background: #627c83; /* Calypso --theme-submenu-background-color */
+}
+
+/**
+ * Site Card
+ */
+.admin-color-ocean  #adminmenu li.toplevel_page_site-card {
+	border-bottom: 1px solid #627c83; /* Calypso --theme-submenu-background-color */
+	border-top: 1px solid #627c83; /* Calypso --theme-submenu-background-color */
+}
+
+.admin-color-ocean .site__info .site__title,
+.admin-color-ocean .site__info .site__domain {
+	color: #ffffff !important; /* Calypso --theme-text-color */;
+}
+
+.admin-color-ocean #adminmenu .toplevel_page_site-card:hover div.wp-menu-image,
+.admin-color-ocean #adminmenu .toplevel_page_site-card a:focus div.wp-menu-image {
+	background-color: #9ebaa0; /* Calypso --theme-highlight-color */
+}
+
+.admin-color-ocean .site__info .site__title::after,
+.admin-color-ocean .site__info .site__domain::after {
+	background: linear-gradient(90deg,rgba(115, 142, 150,0),rgba(115, 142, 150,1) 90%); /* Calypso --theme-base-color-rgb */
+}
+
+.admin-color-ocean .site__info > .site__badge {
+	background: #9ebaa0; /* Calypso --theme-highlight-color */
+	color: #fff; /* Calypso --theme-text-color */
+}
+
+/* Sunrise */
+
+/* Masterbar */
+.admin-color-sunrise #wpadminbar {
+	-webkit-box-shadow: inset 0 -1px 0 #be3631;
+    -moz-box-shadow: inset 0 -1px 0 #be3631;
```

Comment:
You have a few spaces instead of tabs in this file. You'll probably want to switch to tabs everywhere so that no one updates all those to tabs next time they save the file.

Reply:
Thanks for noticing! I pasted these lines from chrome... Fixed 1c15084